addons/sale_project_stock/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Sale Project - Sale Stock',
+    'version': '1.0',
+    'description': 'Bridge between Sale Project and Sale Stock modules',
```

Comment:
Technical terms are not user friendly.

```suggestion
    'description': 'Adds a full traceability of inventory operations on the profitability report.',
```